arch/lib/Makefile

```diff
@@ -155,7 +157,7 @@ define cmd_lib_bounds
 	 echo "#define GENERATED_BOUNDS_H"; \
 	 echo ""; \
 	 echo "#define NR_PAGEFLAGS (__NR_PAGEFLAGS)"; \
-	 echo "#define MAX_NR_ZONES (__MAX_NR_ZONES)"; \
+	 echo "#define MAX_NR_ZONES 3"; \
```

Comment:
> For this problem, I don't understand how to put the hard-coded value in
> .config file.

for instance, you added PAGE_OFFSET value in arch/lib/Kconfig. this value would be written in .config after make defconfig.
```
config PAGE_OFFSET
	hex
	default 0xC0000000
```

you can refer this value by using CONFIG_PAGE_OFFSET by #include <generated/autoconf.h>.